lbz/authz/collector.py

```diff
@@ -24,5 +27,8 @@ def dump(self) -> dict:
             }
         }
 
+    def clean(self) -> None:
+        self.possible_permissions = {}
```

Comment:
Shouldn't it also unset the rest of the attributes? I mean `resource_name` and `guest_permissions` 😉